libfwupdplugin/fu-gcab.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2023 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+#include <locale.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#endif
```

Comment:
what do you think about just moving this `fwupdplugin.h`?

Reply:
Hmm, indifferent. In a way I'd rather keep it isolated to avoid putting windowsisms in random files -- i.e. try to keep them to the `-windows.[c|h]` files.

Reply:
okay, I don't feel strongly.